etc/docker/docker_entrypoint.sh

```diff
@@ -92,4 +92,8 @@ echo "OTBR_WEB_OPTS=\"-I $TUN_INTERFACE_NAME -d7 -p 80\"" >/etc/default/otbr-web
 
 /app/script/server
 
-tail -f /var/log/syslog
+for i in 1 2 3
+do
+    tail -f /var/log/syslog
+    sleep 1
+done
```

Comment:
not sure if 3 seconds is enough or not, how about waiting until the file is created?
```suggestion
while ! stat /var/log/syslog; do
    sleep 1
done

tail -f /var/log/syslog
```

Reply:
Yes we could wait indefinitely but how do you feel about the user having to CTRL+c when there is in fact a real error that prevents syslog from starting?

FWIW in my tests a 1 second wait was always more than enough so using 3 feels like a good compromise between functionality and wait for exit in case of an error.

Reply:
it seems ctrl+c will stop the loop from my test.